test/packages/parallel/sql_input/agent/input/input.yml.hbs

```diff
@@ -5,5 +5,5 @@ hosts:
   - {{this}}
 {{/each}}
 driver: {{driver}}
-sql_query: {{sql_query}
-sql_response_format: {{sql_response_format}}
\ No newline at end of file
+sql_query: {{sql_query}}
```

Comment:
There were no tests for this package before this PR :slightly_smiling_face:

Linters didn't detect them because I think we are not checking these handlebars templates. We would need a handlebars parser in go.

Reply:
> There were no tests for this package before this PR

that's true :sweat_smile: 

> Linters didn't detect them because I think we are not checking these handlebars templates. We would need a handlebars parser in go.

:+1: